specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-02-01/managedClusters.json

```diff
@@ -2641,6 +2753,68 @@
       },
       "description": "The time span with start and end properties."
     },
+    "RunCommandRequest": {
```

Comment:
>RunCommandRequest [](start = 5, length = 17)

Are there any properties required in the request? You can define the required ones in "required" property

Reply:
fixed.